packaging/linux/run_keybase

```diff
@@ -216,6 +228,12 @@ KEYBASE_NO_GUI="${KEYBASE_NO_GUI:-0}"
 KEYBASE_NO_KBFS="${KEYBASE_NO_KBFS:-0}"
 KEYBASE_AUTOSTART="${KEYBASE_AUTOSTART:-0}"
 KEYBASE_KILL="${KEYBASE_KILL:-0}"
+
+if [[ -z "$1" ]]; then
```

Comment:
Should be `-n`?

Reply:
Actually better to check `$#` anyway. Though I think this being in this location messes with the arguments `-h`, etc. that are interpreted as the url here. Also, I think it should be ``$@`` instead, see https://github.com/koalaman/shellcheck/wiki/SC2068.

Reply:
@heronhaye Does `if [[ $# -eq 1 && -n "$1" && "$1" == web+stellar:* ]]; then
` work for you?

Reply:
Oh, can we ensure there will always be a single link? i.e., %u instead of %U?

Reply:
If so, that works for me. (Ignore my other comment). Can likely be `if [[ $# -eq 1 && "$1" == web+stellar:* ]]; then `, but lmk when pushed and I'll try it out. Thanks

Reply:
Oh, weird, I don't know how you could get multiple web+stellar URLs at once. We don't have to support that. Moved over to %u as you suggest.